README.md

```diff
@@ -1,5 +1,6 @@
 # Network SDK
 
+![](https://github.com/karhoo/karhoo-android-sdk/workflows/Master%20Branch%20CI/badge.svg) 
 ![](https://github.com/karhoo/karhoo-android-sdk/workflows/Develop%20Branch%20CI/badge.svg)
```

Comment:
Probably you need to remove the badge for the develop branch (workflow)

Reply:
Sure. Will remove that.